chat/rich-links/src/RichLink/RichLinkBadge/RichLinkBadge.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { Body } from '@leafygreen-ui/typography';
+
+import { useDarkMode } from '../../../../../packages/leafygreen-provider/src/DarkModeContext';
+
+import { badgeVariants, baseStyles, labelStyles } from './RichLinkBadge.styles';
+import { RichLinkBadgeProps } from './RichLinkBadge.types';
+
+const RelativeContainer = ({
+  children,
+  color,
+}: {
+  children: React.ReactNode;
+  color?: string;
+}) => {
+  return <div style={{ position: 'relative', color }}>{children}</div>;
+};
+
+export interface RichLinkBadgesProps {}
+
+export const RichLinkBadges = (props: RichLinkBadgesProps) => {
```

Comment:
Is this used or exported anywhere?

Reply:
Good catch - leftover from early debugging